.phpstan/stubs/Exporter.file

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Exporter;
+
+use Sonata\Exporter\Handler;
+use Sonata\Exporter\Source\SourceIteratorInterface;
+use Sonata\Exporter\Writer\CsvWriter;
+use Sonata\Exporter\Writer\JsonWriter;
+use Sonata\Exporter\Writer\XlsWriter;
+use Sonata\Exporter\Writer\XmlWriter;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+@trigger_error(
+    'The '.__NAMESPACE__.'\Exporter class is deprecated since version 3.1 and will be removed in 4.0.'.
+    ' Use Sonata\Exporter\Exporter instead',
+    E_USER_DEPRECATED
+);
+
+/**
+ * NEXT_MAJOR: remove this class, and the dev dependency.
+ *
+ * @deprecated since sonata-project/core-bundle 3.19, to be removed in 4.0.
+ */
+class Exporter
+{
+    /**
+     * @param string $format
+     * @param string $filename
+     *
+     * @throws \RuntimeException
+     *
+     * @return StreamedResponse
+     */
+    public function getResponse($format, $filename, SourceIteratorInterface $source)
+    {
+        switch ($format) {
```

Comment:
Do we need the method body?

Reply:
You're right, we don't need it.

Reply:
And should we add a `NEXT_MAJOR: Remove this stub when XXX file doesn't need the CoreBundle dependency` to the three stub we added @vladyslavstartsev  ?

In the next major we'll fully remove the CoreBundle dependency, so these stub won't be necessary isn't it ?

Reply:
everything is correct @VincentLanglet @core23